t/schema.t

```diff
@@ -0,0 +1,39 @@
+use strict;
+use warnings FATAL => 'utf8';
+use utf8;
+use open ':std', ':encoding(UTF-8)'; # force stdin, stdout, stderr into utf8
+
+use Data::UUID;
+use Path::Tiny;
+use Test::Conch;
+use Test::Deep;
+use Test::Memory::Cycle;
+use Test::More;
+use Test::Warnings ':all';
+use JSON::Validator;
+
+
+my $json_schema = JSON::Validator->new->schema('http://json-schema.org/draft-04/schema#')->schema->data;
+
+my $t = Test::Conch->new;
+$t->load_fixture('legacy_datacenter');
+
+$t->get_ok('/ping')->status_is(200)
+	->json_is('/status' => 'ok')
+	->header_isnt('Request-Id' => undef)
+	->header_isnt('X-Request-ID' => undef);
+
+$t->get_ok('/schema/response/Login')
+       ->status_is(200)
+       ->json_schema_is($json_schema)
+       ->json_is('/type' => 'object')
```

Comment:
You may find `json_cmp_deeply` more convenient to use, to test more of the data structure all in one go. I found json_is to be really fiddly in having to specify just one bit at a time, leading to lots of little tests.

Reply:
I'm gonna leave these as-is since they're simple. I have used it for an expanded example.